ci-operator/config/openshift/origin/openshift-origin-master.yaml

```diff
@@ -267,6 +267,11 @@ tests:
   steps:
     cluster_profile: packet
     workflow: baremetalds-e2e-virtualmedia
+- as: e2e-metal-ipi-serial-ipv4
```

Comment:
ipv4 is assumed to be the default, it shouldn't be part of the test name to be consistent with other platforms

Reply:
Thanks @stbenjam, you are right. I changed that way.